go/base/Dockerfile.tmpl

```diff
@@ -24,6 +24,9 @@ RUN apt-get -o Acquire::Check-Valid-Until=false update -y --no-install-recommend
             python3-venv \
             python3-pip \
             python3 \
+{{- end }}
+{{- if (eq .DEBIAN_VERSION "11")}}
+            libc6=2.31-13+deb11u4 --allow-downgrades \
```

Comment:
this dependency is needed when installing `build-essentials`: https://packages.debian.org/bullseye/devel/build-essential